Sources/Basics/JSON+Extensions.swift

```diff
@@ -78,23 +78,35 @@ extension JSONDecoder {
 
 extension JSONEncoder {
     public static func makeWithDefaults(prettified: Bool = true, dateEncodingStrategy: DateEncodingStrategy = .safeISO8601) -> JSONEncoder {
+        Self.makeWithDefaults(sortKeys: prettified, prettyPrint: prettified, escapeSlashes: !prettified, dateEncodingStrategy: dateEncodingStrategy)
```

Comment:
These defaults make sense to me based on `prettified`.  That makes for a convenient shorthand, I think.